### DISCUSSION TITLE
Process stop at : INFO [alembic.runtime.migration] Will assume non-transactional DDL.

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#3787] [General]
# Bug Report
Process stop at : INFO [alembic.runtime.migration] Will assume non-transactional DDL.

## Description
When I launch open-webui with Start windows.bat

the process stop at step : INFO [alembic.runtime.migration] Will assume non-transactional DDL. even if I wait lot of minutes  

**Bug Summary:**
[Provide a brief but clear summary of the bug]

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Open WebUI Version:** :  v0.3.8
- **Ollama (if applicable):**   v0.2.1
- 
- **Operating System:**  Windows server datacenter 2022
- **Browser (if applicable):** Chrome, Edge 

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [X ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots
![image](https://github.com/open-webui/open-webui/assets/165768738/96d77b4b-5f55-4a40-9df5-dce747d24091)

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by silentoplayz** [comment]
This is not a bug. This line is informational, and it means that Alembic will assume that the Data Definition Language (DDL) operations (e.g., creating tables, indices, etc.) will not be executed within a transaction. In other words, when Alembic applies migrations to the database, it will not wrap the DDL operations in a transaction. This means that if any of these operations fail, the database will not roll back to its previous state, and the migration process will continue.

By assuming non-transactional DDL, Alembic is allowing the database to execute these operations directly, without creating a transaction that could be rolled back. This is often the default behavior for many databases, as transactions are not always supported for DDL operations.

In general, this message is just an informational notification, and it doesn't require any specific action from you. Alembic is designed to handle these situations, and the migration will continue as usual.

Related: https://github.com/open-webui/open-webui/pull/3595

**Comment by zouzeTG** [comment]
thank's for your response, but the process seems stopped at this step and I don't have any other step started like below 
INFO:     Started server process [14860]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.4:8084 (Press CTRL+C to quit) 
This step don't appear 

that's why http://127.0.0.4:8084 don't work !

**Reply by silentoplayz** [reply]
We've converted your issue to a discussion as we don't believe there's a technical problem at play. It's possible that the issue you're experiencing is due to human error. To help troubleshoot, we recommend reviewing Open WebUI documentation at https://docs.openwebui.com/. If you're still having trouble, feel free to provide more details, as I'm sure there will be people that will do their best to assist you.
